docs - various edits (#72185)
metabase/metabase: docs/embedding/introduction.md

**When to use SSO**: You want to offer multi-tenant, self-service analytics, or you want to include the query builder, AI chat, drill-through, or a collection browser.

> If you're building a SaaS product and embedding analytics for multiple customers, you can keep customer data isolated with [Tenants](./tenants.md).

Author: ```suggestion
If you're building a SaaS product with embedded analytics for multiple customers, you can keep customer data isolated with [Tenants](./tenants.md).
```

\*\* Requires a [Pro and Enterprise](https://www.metabase.com/pricing/) plan for any embedding type.

> ### Should you use Modular embedding or the SDK?

Author: ```suggestion
### Should you use the Modular embedding SDK?
```

If your app uses React, you can go with the modular embedding SDK, but you don't need to.

> The modular embeds that you can set up in the in-app wizard are built on top of the Modular embedding SDK. Using the SDK just gives you slightly more customization (see the table above), but your app has to use React. You can always start with modular embedding, then move to the SDK if you really need that extra customization. Both support SSO and Guest embeds.

Author: ```suggestion
The modular embeds that you can set up in the [in-app wizard](./modular-embedding.md) are built on top of the Modular embedding SDK. Using the SDK just gives you slightly more customization (see the table above), but your app has to use React. You can always start with modular embedding, then move to the SDK if you really need that extra customization. Both support SSO and Guest embeds.
```

> The modular embeds that you can set up in the in-app wizard are built on top of the Modular embedding SDK. Using the SDK just gives you slightly more customization (see the table above), but your app has to use React. You can always start with modular embedding, then move to the modular embedding SDK if you really need that extra customization. Both support SSO and Guest embeds.

Author: ```suggestion
The modular embeds that you can set up in the in-app wizard are built on top of the Modular embedding SDK. Using the SDK just gives you slightly more customization (see the table above), but your app has to use React. You can always start with modular embedding, then move to the SDK if you really need that extra customization. Both support SSO and Guest embeds.
```

> The modular embeds that you can set up in the in-app wizard are built on top of the Modular embedding SDK. Using the SDK just gives you slightly more customization (see the table above), but your app has to use React. You can always start with modular embedding, then move to the modular embedding SDK if you really need that extra customization. Both support SSO and Guest embeds.

Author: Added a link for context.
